lib/ApiResponse.php

```diff
@@ -11,9 +11,24 @@
  */
 class ApiResponse
 {
+    /**
+     * @var array|CaseInsensitiveArray|null
+     */
     public $headers;
+    
+    /**
+     * @var string
+     */
     public $body;
+
+    /**
+     * @var array|null
+     */
     public $json;
+
+    /**
+     * @var int 
```

Comment:
```suggestion
     * @var int
```

The trailing whitespace is causing the linter to fail.

Reply:
Oops, fixed.